scripts/travis_checker.sh

```diff
@@ -37,7 +37,7 @@ go_dirs="${tmpdir}/go_dirs.txt"
 dirnames < "${go_files}" | sort -u -t/ > "${go_dirs}"
 
 echo "Running go test..."
-if go test ./...
+if go test -v ./... -count=1
```

Comment:
```suggestion
if go test -v -count=1 ./...
```
So as to conform to the typical UNIX command line convention (positional arguments come after optional arguments).

Reply:
sure.